cmd/init.go

```diff
@@ -211,6 +217,17 @@ func (o *initOptions) run(cmd *cobra.Command, args []string) error {
 
 	fmt.Println("GitOps worktree set up successfully!")
 
+	// Create secrets directory
+	secretsDir := gitopsConfig + "/secrets"
+	if err := os.MkdirAll(secretsDir, 0755); err != nil {
```

Comment:
Can you make this `0660` or something so it is not readable by the world?

Reply:
Yeah changed it everywhere https://github.com/bitswan-space/bitswan-gitops-cli/pull/38/commits/56d39f0923d7ed97ac10839c6e597e93f77a452b